pandas/core/generic.py

```diff
@@ -5076,6 +5076,9 @@ def pipe(self, func, *args, **kwargs):
         -----
         `agg` is an alias for `aggregate`. Use the alias.
 
+        Pandas operations generally exclude NaNs. For example, `agg(np.nanmedian)`,
+        `agg(np.median)`, and `agg('median')` will return the same result.
```

Comment:
I think you'll need double backticks here to get this to display as code, e.g.
```
``agg(np.median)``
```
instead of 
```
`agg(np.median)`
```

Reply:
Yes sure, fixed that. I was hesitant to use ` `` ` because I thought during docstring validation anything b/w ` `` ` would executed.